CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### Added
+
+- [TAN-830] Added Claude Instant 1 LLM in the backend.
```

Comment:
That's probably too difficult for anyone non-technical to make sense of, so I'd either put it in the `Technical` section, or wait to add it to the changelog until it actually does something in the app.

Reply:
I wanted to put technical (or just leave it out indeed), but then I didn't see any past entries with technical, which confused me. I'll change it.